### ISSUE TITLE
Dedicated Android TV Version

### ISSUE THREAD

**Issue by kfrack3** [z-huang/music#1477]
### Checklist

- [X] I've checked that there is no other issue about this feature request.
- [X] This issue contains only one feature request.
- [X] The title of this issue accurately describes the feature request.

### Feature description

I want the app to have a standalone version for Android TV, with accurate remote controls, and the UI actually made for the TV.

### Why do you want this feature?

So we can have a dedicated music app on the TV.

**Comment by Nolli20** [comment]
Or just a cast

**Comment by serrq** [comment]
Hey, also the Android Auto one...

**Comment by kfrack3** [comment]
> Or just a cast

I would prefer the standalone version as casting is limited, but it's not a totally bad idea.

Also an Android Auto version would be great, even if I don't have a car haha.

**Comment by TruongTanLuc201270** [comment]
It work android tv but cannot backup and restore, aslo need update app icon for android tv.
![Screenshot_20240901-101945](https://github.com/user-attachments/assets/3935c67b-702a-4303-9d02-a148b9b26267)
![Screenshot_20240901-101940](https://github.com/user-attachments/assets/eb9eec92-3687-4727-9a6a-9e5154f2eca2)
![Screenshot_20240901-102028](https://github.com/user-attachments/assets/4855024b-0eaf-48c4-b600-f6dbd1d37972)

**Comment by kfrack3** [comment]
> It work android tv but cannot backup and restore, aslo need update app icon for android tv. ![Screenshot_20240901-101945](https://private-user-images.githubusercontent.com/145727083/363455361-3935c67b-702a-4303-9d02-a148b9b26267.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjUxNjk1ODcsIm5iZiI6MTcyNTE2OTI4NywicGF0aCI6Ii8xNDU3MjcwODMvMzYzNDU1MzYxLTM5MzVjNjdiLTcwMmEtNDMwMy05ZDAyLWExNDhiOWIyNjI2Ny5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwOTAxJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDkwMVQwNTQxMjdaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1jNjBjYjZjZjEyNDQ3Y2FhOTViNDFkNThkNTY1NjA3YTY5YTIwMmRkOWQ0ZGQzZDIxMTZmNGRiZGUyNTZlMGNjJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.KsIxTRUp0GUDX4NCyI5DLGFJgHVu7UefdBd8nLeoXCQ) ![Screenshot_20240901-101940](https://private-user-images.githubusercontent.com/145727083/363455366-eb9eec92-3687-4727-9a6a-9e5154f2eca2.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjUxNjk1ODcsIm5iZiI6MTcyNTE2OTI4NywicGF0aCI6Ii8xNDU3MjcwODMvMzYzNDU1MzY2LWViOWVlYzkyLTM2ODctNDcyNy05YTZhLTllNTE1NGYyZWNhMi5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwOTAxJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDkwMVQwNTQxMjdaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT02MGFkZWVkM2UzNzliNDAyN2E4OTY2MWJkYTEyY2M1MzI0ZGU0NGJmNGYyNWI1NDQzMjhiODllZjU4MmViNjExJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.-cugIosAT2M-EgFFuoEFB6tKJIPG6UNZe4-FIQi8WQk) ![Screenshot_20240901-102028](https://private-user-images.githubusercontent.com/145727083/363455368-4855024b-0eaf-48c4-b600-f6dbd1d37972.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjUxNjk1ODcsIm5iZiI6MTcyNTE2OTI4NywicGF0aCI6Ii8xNDU3MjcwODMvMzYzNDU1MzY4LTQ4NTUwMjRiLTBlYWYtNDhjNC1iNjAwLWY2ZGJkMWQzNzk3Mi5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwOTAxJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDkwMVQwNTQxMjdaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1kY2Q0ODNkNGZiNDE3MjBiOTkzYzdiYjkzMTQzMTgxMzU3Mzk3ZTFkNTZhNWQ1MGE5MDRjMTZiMjRlNzlkMGQ5JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.cuaREQ45emuagA9nTEYjwobl1e_NvygkJ7jaDpDfrpQ)

Yeah, but it doesn't work as well as you'd like so I would like one with the UI better suited for the TV.
